Kickstarter-iOS/Views/Controllers/SetYourPasswordViewControllerTests.swift

```diff
@@ -0,0 +1,34 @@
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+import Prelude
+import XCTest
+
+final class SetYourPasswordViewControllerTests: TestCase {
+  override func setUp() {
+    super.setUp()
+
+    AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    UIView.setAnimationsEnabled(false)
+  }
+
+  func testView() {
+    combos([Language.en], [Device.phone4_7inch, Device.phone5_8inch, Device.pad]).forEach {
+      language, device in
+      withEnvironment(language: language) {
+        let controller = SetYourPasswordViewController.instantiate()
+        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: controller)
+
+        self.scheduler.run()
+
+        FBSnapshotVerifyView(parent.view, identifier: "lang_\(language)_device_\(device)")
```

Comment:
Also these tests aren't accurate, because you need to configure the view controller with `configuredWith` before the snapshot is taken. Try to find a similar view controller that's tested.
If that doesn't work, use `allowLayoutPass` or `schedule.advance(by)` instead of `run`.

Basically the goal is to test the VC as we see it in the app, so we're missing button text, textfield labels, context label text.

Try to include those 👍🏿